package.json

```diff
@@ -32,11 +32,12 @@
     "dbus2js": "./bin/dbus2js.js"
   },
   "dependencies": {
+    "abstract-socket": "^2.0.0",
```

Comment:
it put abstract-socket in your main dependencies, while it exists in `optionalDependencies` already.

Reply:
oops, my fault, tried to use web based conflict resolution for the first time

Reply:
I added you as collaborator, feel free to merge trivial changes without review

Reply:
Cool, let's see how it goes.